trafilatura/utils.py

```diff
@@ -313,6 +313,9 @@ def sanitize_tree(tree):
 @lru_cache(maxsize=1024)
 def trim(string):
     '''Remove unnecessary spaces within a text string'''
+    if string is None:
```

Comment:
This is superfluous considering the try/except AttributeError below

Reply:
yes i added it partially for the debugger and partially to be explicit. can remove though